hwilib/psbt.py

```diff
@@ -86,6 +88,12 @@ def __init__(self) -> None:
         self.hd_keypaths: Dict[bytes, KeyOriginInfo] = {}
         self.final_script_sig = b""
         self.final_script_witness = CTxInWitness()
+        self.tap_key_sig = b""
+        self.tap_script_sigs: Dict[Tuple[bytes, bytes], bytes] = {}
+        self.tap_scripts: Dict[Tuple[bytes, int], Set[bytes]] = {}
+        self.tap_keypaths: Dict[bytes, Tuple[Set[bytes], KeyOriginInfo]] = {}
```

Comment:
42963840ebe71e8553681e5ce80b84de8ba939ec : Same suggestion as on the Bitcoin Core side, to call this `tap_bip32_paths` or some other term that's not ambigious against keypath spend.

Reply:
Done